src/extension.ts

```diff
@@ -441,8 +441,11 @@ class ExtensionManager implements vscode.Disposable {
 
     // Don't configure if the current project is not CMake based (it doesn't have a CMakeLists.txt in the sourceDirectory).
     const sourceDirectory: string = cmt.workspaceContext.config.sourceDirectory;
-    const expandedSourceDirectory: string = util.lightNormalizePath(await expandString(sourceDirectory, { vars: optsVars }));
-    if (await fs.exists(expandedSourceDirectory + "/CMakeLists.txt")) {
+    let expandedSourceDirectory: string = util.lightNormalizePath(await expandString(sourceDirectory, { vars: optsVars }));
+    if (path.basename(expandedSourceDirectory).toLocaleLowerCase() !== "cmakelists.txt") {
+      expandedSourceDirectory = path.join(expandedSourceDirectory, "/CMakeLists.txt");
```

Comment:
`path.join` shouldn't need you to add the directory separator in `"/CMakeLists.txt"`